package.json

```diff
@@ -1,15 +1,8 @@
 {
   "name": "kibana",
-  "description": "Kibana is an open source (Apache Licensed), browser based analytics and search dashboard for Elasticsearch. Kibana is a snap to setup and start using. Kibana strives to be easy to get started with, while also being flexible and powerful, just like Elasticsearch.",
```

Comment:
Can revert these changes? Doesn't seem necessary

Reply:
Absolutely. Not even sure when that happened.

Reply:
I'm still seeing changes here. Try just copying the file from master (https://raw.githubusercontent.com/elastic/kibana/master/package.json) into your copy and commit/push that up